### PR TITLE
Ft/header-section

### PR DESCRIPTION
**Pull request by iamzilfa**
# Description

Done working on the header section

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent

### CODE REVIEW

File: components/sections/Header.jsx [hunk 1]
```diff
@@ -1,11 +1,54 @@
...
-        <Wrapper styles=''>
-      <h1 className="text-8xl font-medium font-sans">Dacade</h1>
- </Wrapper>
-    );
+function Header() {
+  return (
+    <Wrapper>
+      <div className="flex flex-col mt-12 md:flex-row  md:gap-x-13 lg:gap-x-81">
+        <h1 className="sm:text-2xl sm:leading-8 sm:tracking-thin font-medium text-xl leading-7 tracking-thiner min-w-[100px]">
+          Ape Unit
```
Comment (NNesta) [inline components/sections/Header.jsx:9]: remove those arbitrally values on min-w-[100px] and make sure it is in rem

**Review by NNesta — CHANGES_REQUESTED** [review]

File: components/sections/Header.jsx [hunk 2]
```diff
@@ -1,11 +1,54 @@
...
-function Header(props) {
-    return (
-        <Wrapper styles=''>
-      <h1 className="text-8xl font-medium font-sans">Dacade</h1>
- </Wrapper>
-    );
+function Header() {
+  return (
+    <Wrapper>
+      <div className="grid mt-12 tablet:grid-cols-[250px_minmax(0,_1fr)] desktop:grid-cols-3">
```
Comment (NNesta) [inline components/sections/Header.jsx]: Remove this arbitrally value of grid-cols-[250-x_minmax(0,1fr)]

**Review by NNesta — CHANGES_REQUESTED** [review]

**Review by NNesta — APPROVED** [review]
Well done

**Review by esther921 — APPROVED** [review]
LGTM

**Pull request merged**